.drone/drone.yml

```diff
@@ -122,6 +122,56 @@ steps:
 depends_on:
   - Test
 
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+---
+kind: pipeline
+type: docker
+name: Deploy-To-Deployment-Tools 
+platform:
+  os: linux
+  arch: amd64
+trigger:
+  ref:
+    - refs/heads/main
+
+image_pull_secrets:
+  - dockerconfigjson
+
+steps:
+  - name: put image tag in a file
+    image: alpine
+    commands:
+      - apk update && apk add git
+      - echo "grafana/agent:$(sh ./tools/image-tag)" > .image-tag
+  - name: Update Deployment Tools 
+    image: us.gcr.io/kubernetes-dev/drone/plugins/updater
+    settings:
+      config_json: |- 
+        {
+          "destination_branch": "master",
+          "pull_request_branch_prefix": "cd-agent",
+          "pull_request_enabled": false,
+          "pull_request_team_reviewers": [
+            "agent-squad"
+          ],
```

Comment:
We can probably take this part out, but I think it's useful to have it in for when we want to copy this for non-dev flows